internal/service/searchindex/resource_search_index_test.go

```diff
@@ -38,7 +39,12 @@ func TestAccSearchIndex_basic(t *testing.T) {
 				ImportStateVerify: true,
 			},
 		},
-	})
+	}
+}
+
+func TestAccSearchIndex_basic(t *testing.T) {
+	basicCase := basicTestCase(t)
```

Comment:
Uncertain about the convention here. Inline the call or keep on two lines? WDYT?

Reply:
inline is ok for me

Reply:
Cool. Choosing inline then 👍